WorldTrotter/ConversionViewController.swift

```diff
@@ -23,12 +24,18 @@ class ConversionViewController: UIViewController{ //, UITextFieldDelegate {
     //  (hint-use Documentation to find a NSCharacterSet collection for letters, and a String method that finds a range using a NSCharacterSet)
     func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
         
+        
+        let letters = CharacterSet.letters
+        let range = string.rangeOfCharacter(from: letters)
```

Comment:
No need for the extra assignment, just 
string.rangOfCharacter(from: CharacterSet.letters)